importer/handlers/shapefile/handler.py

```diff
@@ -151,15 +152,28 @@ def create_ogr2ogr_command(files, original_name, ovverwrite_layer, alternate):
         )
         layers = ogr.Open(files.get("base_file"))
         layer = layers.GetLayer(original_name)
-        additional_option = (
+
+        if not files.get("cpg_file") and files.get("cst_file"):
```

Comment:
Can you please also add a test here: https://github.com/GeoNode/geonode-importer/blob/3b07221ef3f5dbae50d1e358ddd839c747d0ec77/importer/handlers/shapefile/tests.py#L18
To be sure that behaves as expected and that the command is generated as expected